js/source/tile_pyramid.js

```diff
@@ -271,6 +273,10 @@ TilePyramid.prototype = {
             var zoom = coord.z;
             var overscaling = zoom > this.maxzoom ? Math.pow(2, zoom - this.maxzoom) : 1;
             tile = new Tile(wrapped, this.tileSize * overscaling);
+            // esri/chelm
+            if (this.index) {
+              tile.parentId = this.indexSearch(coord.id);
```

Comment:
When a tile is added to the map, we search the pyramid's index for a "parentId". A parentId should only exist when a requested tile is not found in the tile index. The index is searched recursively until the next lowest zoom level tile is found.